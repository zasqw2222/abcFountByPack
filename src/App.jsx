import React, { Component } from 'react'
import { Layout, Menu, Breadcrumb } from 'antd'
const { Header, Content, Footer } = Layout
class App extends Component {
    constructor(props) {
        super(props)
    }
    render() {

        return (
            <div className="App">
                <Layout>
                    <Header style={{ position: 'fixed', zIndex: 1, width: '100%' }}>
                        <div className="logo" />
                        <Menu
                            theme="dark"
                            mode="horizontal"
                            defaultSelectedKeys={['2']}
                            style={{ lineHeight: '64px' }}
                        >
                            <Menu.Item key="1">测试首页</Menu.Item>

                        </Menu>
                    </Header>
                    <Content style={{ padding: '0 50px', marginTop: 64 }}>
                        <div style={{ background: '#fff', padding: 24, minHeight: 380 }}>
                            {
                                this.props.children
                            }
                        </div>
                    </Content>
                    <Footer style={{ textAlign: 'center' }}>
                        Ant Design ©2018 Created by Ant UED
    </Footer>
                </Layout>
            </div>
        )
    }
}
export default App