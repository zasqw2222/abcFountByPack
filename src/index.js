export { default as Test } from './component/Test'
export { default as Foo } from './component/Foo'