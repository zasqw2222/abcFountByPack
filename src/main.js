import React from 'react'
import { Button } from 'antd'
import styled from 'styled-components'
const Text = styled.div`
  color: red
`

function Test () {
  return <div>
    <Button>123</Button>
    <Text>123123</Text>
  </div>
}

export default Test