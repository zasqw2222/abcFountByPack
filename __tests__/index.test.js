import React from 'react'
import { Test } from '../src/index'
import renderer from 'react-test-renderer'

test('Test changes the class when hovered', () => {
  const component = renderer.create(
    <Test>Facebook</Test>,
  );
  let tree = component.toJSON();
  expect(tree.type).toBe('div')
});