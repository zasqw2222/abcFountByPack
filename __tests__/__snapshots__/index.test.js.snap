// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test changes the class when hovered 1`] = `
<div>
  <button
    className="ant-btn"
    onClick={[Function]}
    type="button"
  >
    <span>
      123
    </span>
  </button>
  <div
    className="src__Text-smolzl-0 hZFWxk"
  >
    123123
  </div>
</div>
`;
